src/code/code_80097A00.c

```diff
@@ -259,7 +273,13 @@ u8 Inventory_DeleteEquipment(PlayState* play, s16 equipment) {
     return equipValue;
 }
 
-void Inventory_ChangeUpgrade(s16 upgrade, s16 value) {
-    gSaveContext.inventory.upgrades &= gUpgradeNegMasks[upgrade];
-    gSaveContext.inventory.upgrades |= value << gUpgradeShifts[upgrade];
+/**
+ * Changes the current upgrade
+ * 
+ * @param type Type of upgrade (uses `UpgradeType` enum).
+ * @param value Level to change upgrade to (plain number, valid range depends on `upgrade`).
```

Comment:
```suggestion
 * @param upgradeType Type of upgrade (uses `UpgradeType` enum).
 * @param value Level to change upgrade to (plain number, valid range depends on `upgradeType`).
```
Also I'm not a fan of writing things like "upgradeType Type of upgrade", it's just filling the screen with redundancy?

Reply:
The main point of this is to say which enums to use.

Reply:
Yeah so `@param upgradeType uses `UpgradeType` enum.` is enough?

Reply:
The other thing is, it may sound redundant in certain cases, but sometimes (arguably most of the other arguments in here) there are usage details that are worth specifying, and the simplest way to do that is to just list all the params where the slightest confusion may arise.